.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
@ashygee You can remove this file because you already have a `prettier.config.js`